src/nifake/metadata/functions.py

```diff
@@ -1510,21 +1510,22 @@
             {
                 'direction': 'in',
                 'enum': None,
-                'name': 'inputArrayOfFloats',
-                'type': 'ViReal64[]',
+                'name': 'inputArrayOfIntegers',
+                'type': 'ViInt16[]',
                 'documentation': {
-                    'description': 'Array of floats',
+                    'description': 'Array of integers. Optional. If passed in then size must match that of inputArrayOfFloats.',
                 },
-            },
+            },        
             {
                 'direction': 'in',
                 'enum': None,
-                'name': 'inputArrayOfIntegers',
-                'type': 'ViInt16[]',
+                'name': 'inputArrayOfFloats',
```

Comment:
Why did you flip the parameters? Is there some reason I'm missing where only the first one can be optional?